windows-iotcore/TOC.md

```diff
@@ -29,7 +29,7 @@
 ### [Develop foreground applications](develop-your-app/BuildingAppsForIoTCore.md)
 ### [Develop background applications](develop-your-app/BackgroundApplications.md)
 ### [Deploy your app](develop-your-app/AppDeployment.md)
-### [Debug your app](develop-your-app/RemoteDebugging.md)
+### [Deploy your console app](develop-your-app/RemoteDebugging.md)
```

Comment:
i'd like to see more distinction between UWP and console ... at the least, i would use "console application" not "console app" (maybe enforcing a difference between when we talk about app as a UWP app) … also, i'd move this to below "Setup a default app" (that way, all the UWP app items would be lumpbed together).